utilities/scan_functions.py

```diff
@@ -1,4 +1,4 @@
-def scan_field_no_keithley(stop, rate, points):
+def scan_field_no_keithley(stop, rate, points, client, data, bridge):

```

Comment:
Again, please make the change backwards compatible. Either update all the old uses of this function to include these parameters, or make them defaults.